src/python_testing/TC_MOD_1_2.py

```diff
@@ -0,0 +1,180 @@
+            self._verify_supported_mode(supported_mode=supported_mode)
+            # After verifying the struct is correct append the mode value.
+            supported_modes_values.append(supported_mode.mode)
+        logger.info(f"Supported modes values {supported_modes_values}")
+
+        # Currentmode attribute check must be int and must be in the supported modes values.
+        self.step(3)
+        current_mode = await self.read_single_attribute_check_success(endpoint=self.endpoint, cluster=self.cluster, attribute=self.cluster.Attributes.CurrentMode)
+        self._log_attribute('CurrentMode', current_mode)
+        asserts.assert_true(isinstance(current_mode, int), "Current mode is not int")
+        asserts.assert_in(current_mode, supported_modes_values, f"Current mode {current_mode} is not in {supported_modes_values}")
+
+        self.step(4)
+        # DEPONOFF in the Mandatory/Optional Column
+        if await self.attribute_guard(endpoint=self.endpoint, attribute=self.cluster.Attributes.OnMode):
+            on_mode = await self.read_single_attribute_check_success(endpoint=self.endpoint, cluster=self.cluster, attribute=self.cluster.Attributes.OnMode)
+            # On mode can be Nullvalue
+            self._log_attribute("OnMode", on_mode)
+            asserts.assert_true((isinstance(on_mode, int) or on_mode is NullValue),
+                                "Onmode is not int or is not Nullvalue")
+            # Verify that OnMode is in the list of Supported Modes, but if null, cant be verified.
+            if on_mode is not NullValue:
+                asserts.assert_in(on_mode, supported_modes_values, f"Onmode {on_mode} is not in {supported_modes_values}")
+
+        # Validate startup mode (attribute Startup is optional)
+        self.step(5)
+        if await self.attribute_guard(endpoint=self.endpoint, attribute=self.cluster.Attributes.StartUpMode):
+            startup_mode = await self.read_single_attribute_check_success(endpoint=self.endpoint, cluster=self.cluster, attribute=self.cluster.Attributes.StartUpMode)
+            self._log_attribute("StartupMode", startup_mode)
+            asserts.assert_true(isinstance(startup_mode, int), "Startupmode is not int")
+            asserts.assert_in(startup_mode, supported_modes_values,
+                              f"Startupmode {startup_mode} is not in {supported_modes_values}")
+
+        # Verify the string  is str and larger that 1 char.
+        # If is non ci ask the user if can read and understand the string.
+        self.step(6)
+        description = await self.read_single_attribute_check_success(endpoint=self.endpoint, cluster=self.cluster, attribute=self.cluster.Attributes.Description)
+        self._log_attribute("Description", description)
+        asserts.assert_true(isinstance(description, str), "Description attribute is not str")
+        asserts.assert_true(len(description) >= 1, "Description is lower that 1 char.")
+        if not self.is_pics_sdk_ci_only:
```

Comment:
I think maybe let's remove this. I know Tennessee said to add it, but the value we get from verifying this is human-readable is marginal, especially since you could well have strings in different languages. Human readable is VERY subjective.